apps/tb/templates/records/social_history.html

```diff
@@ -88,15 +88,30 @@
   </div>
 
   <div class="row">
-    <div class="form-group">
-      <label class="control-label col-sm-4">
-        Mental health issues
+    <div>
+      <label class="control-label col-sm-5">
+        Mental health issues:
       </label>
-      <p class="form-control-static col-sm-8">
-        [[ item.mental_health_issues ]]
+      <p class="form-control-static col-sm-7">
+        <span ng-show="item.mental_health_issues">
+          some history
```

Comment:
@fredkingham is this debugging markup ?

Reply:
yep, its a bit ambiguous changed it to yes/no, rebased and pushed